cluster/examples/workloads/kubernetes/wordpress/aws/resource-classes/rds-class.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: database.aws.crossplane.io/v1alpha2
+kind: RDSInstanceClass
+metadata:
+  name: standard-mysql
+  annotations:
+    resourceclass.crossplane.io/is-default-class: "true"
+specTemplate:
+  class: db.t2.small
+  masterUsername: masteruser
+  securityGroupRefs:
```

Comment:
```suggestion
  securityGroupIdRefs:
```

Reply:
https://github.com/crossplaneio/stack-aws/blob/60869acd22b339560ad7f94a6595581fb72b8b54/apis/database/v1alpha2/rdsinstance_types.go#L108

Reply:
@soorena776 this was marked as resolved but it looks like it still is invalid. Did you want to correct before merge?

Reply:
sorry, i missed this. fixed now